### ISSUE TITLE
Un flux dédié aux assos suivies/rejointes

### ISSUE THREAD

**Issue by Kilimanj** [simde-utc/portail#130]
Salut, 
J'ai pas vu si c'était déjà prévu, ça serait cool pour que chaque personne puisse regrouper ses infos intéressantes. 

Merci pour le nouveau portail, très cool

**Comment by NastuzziSamy** [comment]
Salut, merci à toi ! 

J'ai pas bien compris, tu veux savoir quelles assos tu suis ou a rejoint ? 

Tu as déjà l'affichage à gauche en fait qui affichent tes assos

**Comment by Kilimanj** [comment]
Non, je veux dire en plus du flux d'actualités où il y a les nouveaux articles de toutes les assos, un flux d'actualité perso qui n'affiche que ceux des assos suivies et/ou rejointes

**Comment by NastuzziSamy** [comment]
Ah yes c'est prévu 👌
